stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2458,9 +2499,9 @@ internal struct _${Self}StorageHeader {
 /// An instance of this class has all `${Self}` data tail-allocated.
 /// Enough bytes are allocated to hold the bitmap for marking valid entries,
 /// keys, and values. The data layout starts with the bitmap, followed by the
-/// keys, followed by the values. This is untyped to allow empty
-/// Buffers to be type-punned. Subclasses add type info to provide deinit and 
-/// more interesting NSSelf impls.
+/// keys, followed by the values.
+//
+// See the top level docs for more details on this type
```

Comment:
“top level docs” is ambiguous.  It might send me searching in the project's `docs/` directory.  Ditto every time this term is used.